src/tokensecurity.js

```diff
@@ -621,7 +621,10 @@ module.exports = function(app, config) {
       if (req.query && req.query.token) {
         token = req.query.token
       } else if (req.headers) {
-        const header = req.headers.authorization
+        let header = req.headers.authorization
```

Comment:
As this is already in two places I think it is worth extracting a function.

How about a test case?